src/mimeshow.jl

```diff
@@ -148,8 +148,10 @@ function _markdown(m::MixedModel)
         push!(rows, newrow)
     end
 
-    re_without_fe = setdiff(
-        mapfoldl(x -> Set(getproperty(x, :cnames)), ∪, m.reterms), coefnames(m)
+    re_without_fe = setdiff!(
+        # why not use `Set`? because ordering can change between releases, while
+        # unique preserves the original order
+        unique(mapfoldl(x -> getproperty(x, :cnames), vcat, m.reterms)), coefnames(m)
```

Comment:
Some quick benchmarking suggests that this is faster with fewer allocations:
```suggestion
        unique!(mapfoldl(x -> getproperty(x, :cnames), append!, m.reterms; init=String[])), coefnames(m)
```
Users could get tens, even hundreds of nanoseconds of their lives back that would otherwise be spent printing to Markdown!